src/payment/strategies/bluesnapv2/bluesnapv2-payment-page-loader.ts

```diff
@@ -0,0 +1,59 @@
+import { createFormPoster } from '@bigcommerce/form-poster';
+
+import { NotInitializedError, NotInitializedErrorType } from '../../../common/error/errors';
+import { PaymentMethodCancelledError } from '../../errors';
+
+import { BlueSnapV2PaymentInitializeOptions } from './bluesnapv2-payment-options';
+
+export default class BlueSnapV2PaymentPageLoader {
+    private _blueSnapV2Options?: BlueSnapV2PaymentInitializeOptions;
+    private _formPoster = createFormPoster();
```

Comment:
instead of creating a form poster, would be better to instantiate further up and pass it via the constructor, so it can be mocked. This would be consistent to how we pass the `requestSender`. You can check any other strategy as an example to see how this is done.

Reply:
We changed the approach so, I removed that part.